### ISSUE TITLE
can not get the select image

### ISSUE THREAD

**Issue by py110** [marcshilling/react-native-image-picker#82]
version:    compile 'com.facebook.react:react-native:0.17.1'
```javascript
 const options = {
            title: '选择图像',
            quality: 0.5,
            maxWidth: 300,
            maxHeight: 300,
            mediaType: 'photo',
            takePhotoButtonTitle: '拍照', // specify null or empty string to remove this button
            chooseFromLibraryButtonTitle: '从图库中选择',
            cancelButtonTitle: '取消',
            storageOptions: {
                skipBackup: true
            },
            noData: true, // photos only - disables the base64 `data` field from being generated (greatly improves performance on large photos)
        }
        UIImagePickerManager.showImagePicker(options, (response) => {
            console.log('Response = ', response);

            if (response.didCancel) {
                console.log('User cancelled photo picker');
            }
            else if (response.error) {
                console.log('UIImagePickerManager Error: ', response.error);
            }
            else if (response.customButton) {
                console.log('User tapped custom button: ', response.customButton);
            }
            else {
                // You can display the image using either:
                //const source = {uri: 'data:image/jpeg;base64,' + response.data, isStatic: true};
                var source = null;
                if (Platform.OS === 'ios') {
                    source = {uri: response.uri.replace('file://', ''), isStatic: true};
                } else {
                    Alert.alert('tt',response.uri);
                    source = {uri: response.uri, isStatic: true};
                    //source = {uri: 'data:image/jpeg;base64,' + response.data, isStatic: true};

                }

                this.setState({
                    avatarSource: source
                });
            }
        });
```
the response.uri in Alert is nothing
![image](https://cloud.githubusercontent.com/assets/13548241/12290934/5e4d66d2-ba1f-11e5-9297-1516ae606d44.png)

**Comment by yfuks** [comment]
Hello,

Since i don't see any imagePicker file in your error stack it's maybe a problem with your react-native implementation of this package. Looks like you try to resume the app or to open the dev 'menu' and that throw an Exception.

You need to debug further.

Check your react-native version.

**Comment by py110** [comment]
version: compile 'com.facebook.react:react-native:0.17.1'

package com.icommunity;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.support.v4.app.FragmentActivity;
import android.util.Log;
import android.view.KeyEvent;
import android.view.View;

import com.aakashns.reactnativedialogs.ReactNativeDialogsPackage;
import com.facebook.react.LifecycleState;
import com.facebook.react.ReactInstanceManager;
import com.facebook.react.ReactRootView;
import com.facebook.react.modules.core.DefaultHardwareBackBtnHandler;
import com.facebook.react.shell.MainReactPackage;
import com.facebook.soloader.SoLoader;
import com.icommunity.updater.UpdateRunnable;
import com.icommunity.wx.pay.Constants;
import com.icommunity.wx.pay.WeChatPackage;
import com.icommunity.wx.pay.WeChatPayPackage;
import com.tencent.mm.sdk.modelbase.BaseReq;
import com.tencent.mm.sdk.modelbase.BaseResp;
import com.tencent.mm.sdk.openapi.IWXAPI;
import com.tencent.mm.sdk.openapi.IWXAPIEventHandler;
import com.oblador.vectoricons.VectorIconsPackage;
import com.tencent.mm.sdk.openapi.WXAPIFactory;
import com.microsoft.codepush.react.CodePush;
import com.imagepicker.ImagePickerPackage;


public class MainActivity extends FragmentActivity/*Activity*/ implements DefaultHardwareBackBtnHandler  {

    private ReactInstanceManager mReactInstanceManager;
    private ReactRootView mReactRootView;
    private ImagePickerPackage mImagePicker;

//    private IWXAPI iwxapi;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        mReactRootView = new ReactRootView(this);
        mImagePicker = new ImagePickerPackage(this);
        new UpdateRunnable(this, new Handler()).force(true).start();
        CodePush codePush = new CodePush("1874e7c5-d018-4645-9e74-e4599102477e", this);
//        iwxapi = WXAPIFactory.createWXAPI(this, Constants.APP_ID, false);
        mReactInstanceManager = ReactInstanceManager.builder()
                .setApplication(getApplication())
                .setJSMainModuleName("index.android")
                .setBundleAssetName("index.android.bundle")
//                .setJSBundleFile(codePush.getBundleUrl("index.android.bundle")) //codepush
                .addPackage(new MainReactPackage()) // codepush
                .addPackage(codePush.getReactPackage())
                .addPackage(new WeChatPackage())
                .addPackage(mImagePicker)
                .addPackage(new VectorIconsPackage())
                .addPackage(new ReactNativeDialogsPackage(this))
                .setUseDeveloperSupport(BuildConfig.DEBUG)
                .setInitialLifecycleState(LifecycleState.RESUMED)
                .build();

        mReactRootView.startReactApplication(mReactInstanceManager, "ZyCommunity", null);

        setContentView(mReactRootView);


    }


    @Override
    public boolean onKeyUp(int keyCode, KeyEvent event) {
        if (keyCode == KeyEvent.KEYCODE_MENU && mReactInstanceManager != null) {
            mReactInstanceManager.showDevOptionsDialog();
            return true;
        }
        return super.onKeyUp(keyCode, event);
    }

    @Override
    public void onBackPressed() {
      if (mReactInstanceManager != null) {
        mReactInstanceManager.onBackPressed();
      } else {
        super.onBackPressed();
      }
    }

    @Override
    public void invokeDefaultOnBackPressed() {
      super.onBackPressed();
    }

    @Override
    protected void onPause() {
        super.onPause();

        if (mReactInstanceManager != null) {
            mReactInstanceManager.onPause();
        }
    }

    @Override
    protected void onResume() {
        super.onResume();

        if (mReactInstanceManager != null) {
            mReactInstanceManager.onResume(this, this);
        }
    }

    @Override
    public void onActivityResult(final int requestCode, final int resultCode, final Intent data) {
        super.onActivityResult(requestCode, resultCode, data);

        mImagePicker.handleActivityResult(requestCode, resultCode, data);
    }

}

**Comment by yfuks** [comment]
Did the app crash ? Do you get an error message ? What is your response ?
```javascript
console.log('Response = ', response);
```

**Comment by py110** [comment]
01-13 19:06:33.450 17779-18716/com.icommunity I/ReactNativeJS: 'Response = ', false

**Comment by marcshilling** [comment]
Are you sure you're on the latest version? `response` should be an object. If you're on an older version, it used to be a boolean.

**Comment by py110** [comment]
resolved! use the ^0.12.4 it is ok,before use ^0.12.2.thanks
